.eslintrc

```diff
@@ -32,7 +32,20 @@
     // custom rules
     "@typescript-eslint/lines-between-class-members": "off",
     "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-throw-literal": "off",
     "@typescript-eslint/no-explicit-any": "error",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "args": "all",
+        "argsIgnorePattern": "^_",
+        "caughtErrors": "all",
+        "caughtErrorsIgnorePattern": "^_",
+        "destructuredArrayIgnorePattern": "^_",
+        "varsIgnorePattern": "^_",
+        "ignoreRestSiblings": true
```

Comment:
I'm on board with all of these rule changes except this one. We already have `lodash`, so we should be using [omit](https://lodash.com/docs/#omit) or [pick](https://lodash.com/docs/#pick) in those cases to avoid unused variables.